### DISCUSSION TITLE
Not all Cargo.toml files are updated

### DISCUSSION THREAD

**Discussion by torokati44** [renovatebot/renovate#23952] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

The documentation states (here: https://docs.renovatebot.com/modules/manager/cargo/#file-matching) that any `Cargo.toml` files anywhere in the repository are updated. This does not seem to be the case.

See for example: https://github.com/ruffle-rs/ruffle/pull/12830
Here, the `anyhow` crate got updated from "1.0.72" to "1.0.75" in `render/naga-pixelbender`, but not in `tests/mocket`.
Another common problematic crate is the frequently updated `serde` - usually updated in `core`, but not in `tests/input-format` or `tests/socket-format`.
In fact, none of the `Cargo.toml` files under `tests` are picked up as "Detected Dependencies" in the project dashboard, but nothing in the docs suggests that this would be the case, or why.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
Tests and examples are ignore by default with the config:base preset

**Reply by torokati44** [reply]
Oh, that would explain it! Thanks for the quick response!
I can't find the docs for `config:base` though...

**Reply by torokati44** [reply]
... that's because it just got renamed to `config:recommended` recently: https://github.com/renovatebot/renovate/commit/f9e3e80e0c9c6a972d847f8740de5016a2bf698a
